### PR TITLE
add output_dir parameter

### PR DESCRIPTION
**Pull request by omerbt**
Added an optional parameter: output_dir. if provided, DeepCell output will be saved in that directory.
https://github.com/vanvalenlab/kiosk-client/issues/65

### CODE REVIEW

**Comment by omerbt** [comment]
I updated the code according to the comments

File: kiosk_client/job.py [hunk 1]
```diff
@@ -90,6 +90,14 @@ def __init__(self, host, filepath, model_name, model_version, **kwargs):
         self.original_name = kwargs.get('original_name', self.filepath)
         self.download_results = kwargs.get('download_results', False)
 
+        self.output_dir = kwargs.get('output_dir', get_download_path())
+        if not os.path.isdir(self.output_dir):
+            raise ValueError("Invalid value for output_dir,"
+                             " %s is not a directory" % self.output_dir)
+        if not os.access(self.output_dir, os.W_OK):
+            raise ValueError("Invalid value for output_dir,"
+                             " %s is not writable" % self.output_dir)
```
Comment (willgraf) [inline kiosk_client/job.py:99]:
```suggestion
            raise ValueError('Invalid value for output_dir,'
                             ' %s is not a directory.' % self.output_dir)
        if not os.access(self.output_dir, os.W_OK):
            raise ValueError('Invalid value for output_dir,'
                             ' %s is not writable.' % self.output_dir)
```

File: kiosk_client/manager.py [hunk 2]
```diff
@@ -113,6 +113,14 @@ def __init__(self, host, job_type, **kwargs):
         self.download_results = kwargs.get('download_results', True)
         self.calculate_cost = kwargs.get('calculate_cost', False)
 
+        self.output_dir = kwargs.get('output_dir', job.get_download_path())
+        if not os.path.isdir(self.output_dir):
+            raise ValueError("Invalid value for output_dir,"
+                             " %s is not a directory" % self.output_dir)
+        if not os.access(self.output_dir, os.W_OK):
+            raise ValueError("Invalid value for output_dir,"
+                             " %s is not writable" % self.output_dir)
```
Comment (willgraf) [inline kiosk_client/manager.py:122]:
```suggestion
        if not os.path.isdir(self.output_dir):
            raise ValueError('Invalid value for output_dir,'
                             ' %s is not a directory.' % self.output_dir)
        if not os.access(self.output_dir, os.W_OK):
            raise ValueError('Invalid value for output_dir,'
                             ' %s is not writable.' % self.output_dir)
```

**Review by willgraf — CHANGES_REQUESTED** [review]
Love the tests! Just one more style-related suggested change. This project uses single quotes (`'`) for strings and double quotes (`"`) for docstrings.

**Comment by omerbt** [comment]
great thanks!

**Review by willgraf — APPROVED** [review]

**Pull request merged**